### ISSUE TITLE
bootstrap issue with yii-multilingual module

### ISSUE THREAD

**Issue by stratosgear** [cniska/yii-app#5]
Hi Cristofer,

I tried to activate the multilingual feature of yii-app and I came across the following issue.

Your MlLanguageMenu widget (from yii-multilingual) extends TbMenu (from yii-boostrap) but in yii-app you are including yiistrap that does not have TbMenu, but rather TbNav.

If I change TbMenu to TbNav, it doesn't complain anymore but the rendering of the languages doesn't render correctly.  I still have to investigate how much different TbMenu is from TbNav, so maybe they are not totally compatible without some further changes.

I didn't know where to mention this issue.  I didn't think reporting it under the yii-mulitlingual project was correct.

Also can you summarize the differences of yii-boostrap vs yiistrap.  Which one are you going to actively develop?

**Comment by cniska** [comment]
I've fixed this issue but updating MlLanguageMenu to use TbNav. Yiistrap is the a new project that should be live soon, think of it as the next generation of Yii-Bootstrap.
